bin/parse-server

```diff
@@ -10,26 +10,28 @@ if (process.env.PARSE_SERVER_OPTIONS) {
 	options = JSON.parse(process.env.PARSE_SERVER_OPTIONS);
 
 } else {
-
-	options.databaseURI = process.env.PARSE_SERVER_DATABASE_URI;
-	options.cloud = process.env.PARSE_SERVER_CLOUD_CODE_MAIN;
-	options.collectionPrefix = process.env.PARSE_SERVER_COLLECTION_PREFIX;
+  options.app                        = {};
```

Comment:
Yeah the singleton providers (mail, cache, database, files) would need to have a map of `app:adapter`. I started implementing it and decided it would be better to get this merged first. The DatabaseProvider already kind of implements this, but I want to refactor it.